webxr-api/frame.rs

```diff
@@ -18,4 +19,6 @@ pub struct Frame {
     /// This is equivalent to the pose of the viewer in native coordinates.
     /// This is the inverse of the view matrix.
     pub transform: TypedRigidTransform3D<f32, Viewer, Native>,
+
+    pub inputs: Vec<InputFrame>,
```

Comment:
Bikeshed: Should this be `events: Vec<Event>`? or `Vec<(InputSourceId, InputEvent)>` or some such?

Reply:
I don't want to think about events right now, trying to implement the minimum possible for integration. Also events can be triggered without an rAF loop, so we can't stuff that in the frame.

Reply:
How about `input_poses: Vec<TypedTransform3D<f32, Input, Native>>` then?

Reply:
It's going to eventually also contain gaze info, and gamepad info

Reply:
At this point I am happy with whatever you want to do. A comment would be nice though!